app/models/service_provider.rb

```diff
@@ -75,6 +75,10 @@ def biometric_ial_allowed?
       IdentityConfig.store.allowed_biometric_ial_providers.include?(issuer)
   end
 
+  def identity_proofing_allowed?
+    ial == 2
+  end
+
```

Comment:
This could be removed, but we have a number of places where we look at the SP.ial to drive business logic. Eventually would like to clean up all those places, but this is a first start.